src/tests/test_jsproxy.py

```diff
@@ -736,3 +736,19 @@ def raises(exc, match=None):
         `);
         """
     )
+
+
+def test_memory_leaks(selenium):
```

Comment:
Maybe

```suggestion
def test_jsproxy_get_iter_memory_leak(selenium):
```
?

Reply:
Here I tested both `iter` and `repr`. I think it could be reasonably practical to update the test to include a bunch of other commands (we can just call various APIs and then assert that nothing leaked at the end). Anyways, I figure the test name and contents can be updated as we increase leak test coverage. Maybe `test_jsproxy_memory_leaks` would be good (though this is already in a file called `test_jsproxy`).